build_scripts/build_usd.py

```diff
@@ -2281,7 +2281,7 @@ def ForceBuildDependency(self, dep):
 
 # Determine list of dependencies that are required based on options
 # user has selected.
-requiredDependencies = [ZLIB, BOOST, TBB]
+requiredDependencies = [BOOST, TBB]
```

Comment:
@dgovil Checkout line 2325.  I think you need to add a check to make sure that `ZLIB` is in `requiredDependencies` now before trying to remove.

Reply:
Ah good catch. I made it check first now as suggested.